src/Entity/Carrinho.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CarrinhoRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: CarrinhoRepository::class)]
```

Comment:
Esse repositoryClass não precisa mais porque já especificamos a entidade no construtor do repository.

```suggestion
#[ORM\Entity]
```

https://github.com/GabrielCirqueira/MS-CODE-PDV-SYMFONY/blob/ec91ccc25f73619c4aef1745d09c2e7cede7d447/src/Repository/CarrinhoRepository.php#L12-L17